build-logic/src/main/groovy/qsl.module.gradle

```diff
@@ -123,7 +123,29 @@ java {
 	withSourcesJar()
 }
 
-// TODO: Configure javadoc options
+javadoc {
+	options {
+		source = "16"
+		encoding = "UTF-8"
+		charSet = "UTF-8"
+		memberLevel = JavadocMemberLevel.PACKAGE
+		links(
+				"https://guava.dev/releases/21.0/api/docs/",
+				"https://asm.ow2.io/javadoc/",
+				"https://docs.oracle.com/javase/8/docs/api/",
+				"https://jenkins.liteloader.com/job/Mixin/javadoc/",
+				"https://logging.apache.org/log4j/2.x/log4j-api/apidocs/"
```

Comment:
Should we add Yarn/QuiltMappings links to this?

Reply:
If it's already generated, yes.

Reply:
It does not look like it is published. This can be resolved once quilt mappings is out

Reply:
ok so its being published, but there doesn't seem to be a forward facing URL to read the docs. I don't know how its automatically done on the fabric repository so ill look into it

Reply:
Crash reports are not going to build without this block, so I'm going to copy it